department-of-veterans-affairs/vets-api
# [7834] Relaxing uniqueness constraint on idme_uuid for Accounts table…

…, so that nil records are possible when sec_id is defined

## Description of change
This PR relaxes the idme_uuid constraint on the `Account` model so that multiple records can be nil, given that a `sec_id` is defined. Previously, there was a bug so that only a single `accounts` record could have a `nil` `idme_uuid`

## Original issue(s)
department-of-veterans-affairs/vets-api/issues/7834

## Things to know about this PR
- To test this, in rails console I ran:
`Account.new(sec_id: 1).save`
`Account.new(sec_id: 2).save`
And confirmed that both were properly saved in the accounts table
Also, `Account.new.save` still failed, as expected, proving that a record still needs with a `sec_id` or `idme_uuid` defined
